api/app/controllers/spree/api/base_controller.rb

```diff
@@ -42,7 +42,11 @@ def permitted_line_item_attributes
       end
 
       def load_user
-        @current_api_user ||= Spree.user_class.find_by(spree_api_key: api_key.to_s)
+        @current_api_user ||= if json_web_token.present?
```

Comment:
We could even use another separate header maybe? Is there some benefit in using `X-Spree-Token` for JWT as well?

Reply:
I figured it would create backward compatibility as well as forward compatibility by allowing both options